### ISSUE TITLE
Tons of Warning message - WARNING: Skipped parameter `{0}` as it is undefined on `{1}`. Set `-D{2}=true` to allow undefined parameters to be injected as environment variables or `-D{3}=[comma-separated list]` to whitelist specific parameter names, even though it represents a security breach or `-D{2}=false` to no longer show this message.

### ISSUE THREAD

**Issue by gintire** [jenkins-infra/helpdesk#3475]
### Service(s)

jenkins.io

### Summary

tl;dr;
Jenkins Monitoring tool (java melody) show master(controller) node generate tons of Warning message.
I wonder it is normal or unexpected.

Monitor Warn message below. (Over 40M warn messages a day)
![image](https://user-images.githubusercontent.com/39238504/228477739-be06047a-3e31-4b93-8deb-468bc32606bb.png)

Env
OS : Windows Server 2016
Jenkins Version : v2.320
Memory : 8G
CPU : Intel processor
Java : Zulu 11

Detail
I monitor there are many warn message on master node. It is related with ParametersAction class (https://github.com/jenkinsci/jenkins/blob/master/core/src/main/java/hudson/model/ParametersAction.java#L314)

An in-house application request results of job regularly. But the application request to all jobs but the "Warning Message" dedicate only job. 

For instance,
![image](https://user-images.githubusercontent.com/39238504/228481825-b1447309-d823-4d5e-9a16-71ffa65ec217.png)

In addition request which hang is changed everyday. 
![image](https://user-images.githubusercontent.com/39238504/228482303-a12b3d4f-217c-4a12-b1b2-c172598a6012.png)

There is no heavy I/O call and occupying disk.

I suppose it occupy memory and threads.

Anyone who know or solve it?
